docs/storage/storage.md

```diff
@@ -155,6 +167,20 @@ xe sr-create host-uuid=<host UUID> type=ext content-type=user name-label="Local
 
 In addition to the two main, rock-solid, local storages (EXT and LVM), XCP-ng offers storage drivers for other types of local storage (ZFS, XFS, etc.).
 
+### Software RAID
+
+Local, with ```mdadm```. Not recommended.
+
+:::tip
+We strongly recommend using a hardware RAID system instead of software RAID.
+:::
+
+:::warning
+**Software RAID storage integration is offered as-is** and does not come with official support.
+
+We do not provide support for issues resulting from the choice of software RAID.
```

Comment:
I know the context is clear as this is about storage, but just in case the sentence is quoted without context, as we do support software raid for the system:

```suggestion
We do not provide support for issues resulting from the choice of software RAID for storage repositories.
```